scripts/release/debian/prepare.sh

```diff
@@ -102,6 +102,9 @@ export DH_OPTIONS=-v
 %:
 ${TAB}dh \$@ --sourcedirectory $source_dir
 
+override_dh_clean:
+${TAB}dh_clean --exclude=__pycache__
```

Comment:
To ensure the pyc files are included in package, as they are excluded by default starting with debhelper version 13.10.10.
See https://github.com/Azure/azure-cli/pull/28888#issuecomment-2099909964

Reply:
This is a more official doc: https://manpages.ubuntu.com/manpages/noble/en/man1/dh_clean.1.html